foxglove_bridge_base/include/foxglove_bridge/websocket_server.hpp

```diff
@@ -974,7 +974,14 @@ template <typename ServerConfiguration>
 inline void Server<ServerConfiguration>::publishParameterValues(
   ConnHandle hdl, const std::vector<Parameter>& parameters,
   const std::optional<std::string>& requestId) {
-  nlohmann::json jsonPayload{{"op", "parameterValues"}, {"parameters", parameters}};
+  // Filter out parameters which are not set.
```

Comment:
The spec currently does not declare what parameter message to expect when requesting a non-existing / deleted parameter. Should it be
```json
{
  "op": "parameterValues", 
  "parameters": []
}
```
or
```json
{
  "op": "parameterValues", 
  "parameters": [{ "name": "/deleted_param", "value": undefined }]
}
```

Reply:
The second example is not valid JSON. There is no `undefined` literal in https://www.rfc-editor.org/rfc/rfc8259

Reply:
Adapted my first comment by removing the `value` field

Reply:
I think the first is better?

Reply:
OK, that's also how it is currently implemented